recipes/ledger/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "ledger" %}
+{% set version = "3.1.3" %}
+{% set sha256 = "b248c91d65c7a101b9d6226025f2b4bf3dabe94c0c49ab6d51ce84a22a39622b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ledger/ledger/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - fix-build.patch
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+    - python >=2.4,<3
```

Comment:
Remove this

Reply:
Remove all four lines? Are these by default included in the build environment? As for Python, the build requires the Python 2.7 libraries

Reply:
No, one line. `python` should be in host and yes, python version should be skipped using a selector. See for eg: https://github.com/conda-forge/numpy-feedstock/blob/master/recipe/meta.yaml